src/models.ts

```diff
@@ -16,19 +16,32 @@ export type TrContext = Omit<TContext, "setLocale">;
 
 type KeyPrefix<T extends string> = T extends "" ? "" : `.${T}`;
 
+interface plural {
+  one?: string;
+  zero?: string;
+  many?: string;
+};
+
+interface gender {
+  female?: string;
+  male?: string;
+};
```

Comment:
It would be easier to merge all dynamic suffixes into a single type. Also, Talkr may return more complex plural keys for certain languages. Here is an updated type:
```suggestion
type DynamicSuffix = Partial<{
  zero: string;
  one: string;
  two: string;
  few: string;
  many: string;
  female: string;
  male: string;
}>;
```